README.md

```diff
@@ -8,21 +8,39 @@ docker build --tag h379wang/kim-model-conversion .
 docker run -it h379wang/kim-model-conversion:latest /bin/bash
 ```
 ----------
+### Get Datasets
+```
+cd kim_cnn
+chmod +x get_data.sh
```

Comment:
As I commented previously in #1, this is based on a really old version version of the Kim CNN code in Castor, since we should not longer be pulling in data this way from a private repo. I would prefer you use the latest version, but ultimately @Impavidity can decide the best course of action.

Reply:
Thanks. @Impavidity What do you think here.

Reply:
We should not be fetching data from @Impavidity - instead, we should be reading data from `Castor-data`.

Reply:
I see. Thanks for the reminder. I will make a commit for the change.